src/xdocs/config_coding.xml

```diff
@@ -3623,6 +3623,200 @@ class TestHashCode {
       </subsection>
     </section>
 
+    <section name="MatchXpath">
+      <p>Since Checkstyle 8.39</p>
+      <subsection name="Description" id="MatchXpath_Description">
+        <p>
+          Evaluates Xpath query and report violation on all matching AST nodes. Current check allows
+          user to implement custom checks using Xpath. If Xpath query is not specified explicitly,
```

Comment:
> Current check 

This sounds funny, like you are saying this wasn't supported by the check before but it is a brand new check. I recommend changing `Current check` => `This check`.

Reply:
done